lib/admin_form_builder.rb

```diff
@@ -0,0 +1,20 @@
+class AdminFormBuilder < GenericFormBuilder
+  STANDARD_FIELDS.each do |method|
+    define_method(method.to_sym) do |field, *args|
+      options, *args = args
+      options ||= {}
+
+      options[:wrapper_html_options] = {class: "form-group"}.merge(options[:wrapper_html_options] || {})
+      options[:class] = Array(options[:class]) + ["form-control", "input-md-4"]
```

Comment:
I think, as much as possible, we could derive sensible defaults from the Model definitions, from type, validation, (even name), but still allow manual overriding if needed.